pkgs/by-name/pi/pint/package.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+, testers
+, pint
+}:
+
+buildGoModule rec {
+  pname = "pint";
+  version = "0.51.0";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = "pint";
+    rev = "v${version}";
+    hash = "sha256-d3KZPPeQJBqdrr81YLusYHc5jLChC1Rf5SYeP/QMeo8=";
+  };
+
+  vendorHash = "sha256-9tJL33Qtu0xAmRnAP0BwM6CIfGs+GEG864e89XrpfLM=";
+
+  subPackages = [ "cmd/pint" ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.version=${version}"
+    "-X=main.commit=${src.rev}"
```

Comment:
src.rev is just `v${version}`. 

```
❯ result/bin/pint version
0.51.0 (revision: v0.51.0)
```

Reply:
It's on purpose, the rev should match a git entity which is the tag for us.

Reply:
Fine for me. :+1: commit revs are annoying to update anyway.